### PR TITLE
feat: add datetime picker beta

### PR DESCRIPTION
**Pull request by benjamin-rocks**

### CODE REVIEW

File: src/components/actionJSForm.js [hunk 1]
```diff
@@ -16,26 +16,12 @@
...
-          if (!error || !error.messages || !error.messages.errors) return [];
-          return error.message.errors.map((inner) => inner.message);
-        });
-
-        B.triggerEvent('onActionError', new Error(messages.join(', ')));
-
-        console.log('Errors: ', messages.join('\n   '));
-        return;
-      }
-
```
Comment (dylanBetty) [inline src/components/actionJSForm.js:31]: Was the removal of the error handling removed deliberately? Is it correct that this is handled via the alert component now? As I see there have been some additions regarding error messages there.

Comment (benjamin-rocks) [reply src/components/actionJSForm.js:31]: Yes, the error handling has been moved there. 

Ideally this should be done inside a helper

**Review by dylanBetty — APPROVED** [review]

File: src/components/actionJSForm.js (hunk 1 above)
Comment (benjamin-rocks) [reply src/components/actionJSForm.js:31]: Created issue PAGE-1845 for the helper

Comment (dylanBetty) [reply src/components/actionJSForm.js:31]: :ok_hand:

**Pull request merged**